src/configure

```diff
@@ -1317,18 +1317,29 @@ elif [ "`uname`" == "Linux" ]; then
     if [ -z "$OPENBLASROOT" ]; then
       failure "Must specify the location of OPENBLAS with --openblas-root option (and it must exist)"
     fi
-    if [ ! -f $OPENBLASROOT/lib/libopenblas.so ]; then
+    if [ -f $OPENBLASROOT/lib/libopenblas.so ]; then
```

Comment:
Can you comment that this is the deb-based install path and the second case is the rpm-based install path for clarity to others?

Reply:
will do